### PR TITLE
api-management

### PR DESCRIPTION
**Pull request by hguerrero**
This is the frontend modifications to show the API management tab and test the services using the apicast microgateway.

### CODE REVIEW

**Comment by hguerrero** [comment]
api-management github repo to fork:

https://github.com/hguerrero/api-management.git

File: index.html [hunk 1]
```diff
@@ -198,6 +201,75 @@ <h3 class="panel-title">Aloha Service</h3>
...
+									<h3 class="panel-title">Aloha Managed</h3>
+									(Vert.x)
+								</div>
+								<div id="aloha-service-managed" class="panel-body">Unauthorized</div>
+							</div>
+						</div>
+					</div>
+					<a id="refresh-apimanagement2" href="#"><span class="pficon pficon-refresh"></span>Refresh Results</a>
+				</div>
+				<img src="images/sso.svg" style="float: right; width: 50%; height: 30em;" />
```
Comment (rafabene) [inline index.html]: Is it supposed to have the same image?

File: index.html [hunk 2]
```diff
@@ -50,7 +50,10 @@
 				<li role="presentation" class="active"><a href="#sso">SSO</a></li>
 			{{/sso}}
 			<li role="presentation"><a href="#browser">Browser as a client</a></li>
-			<li role="presentation"><a href="#apigateway">API Gateway</a></li>
```
Comment (rafabene) [inline index.html]: This tab should come between SSO and "Browser as a client". It was inserted in the middle of the "Invocation patterns".

File: index.html [hunk 3]
```diff
@@ -198,6 +201,75 @@ <h3 class="panel-title">Aloha Service</h3>
 				<img src="images/browser-client.svg" style="float: right; width: 50%; height: 30em;" />
 			</div>
 
+			<!--  Tab API Management -->
```
Comment (rafabene) [inline index.html]: Move here too.

**Review by rafabene — CHANGES_REQUESTED** [review]

**Comment by rafabene** [comment]
The tab still needs to be between SSO and "Browser as a client". It was inserted in the middle of the "Invocation patterns".

**Comment by hguerrero** [comment]
done, moved between "sso" and "browser as client".

**Comment by rafabene** [comment]
Merged at https://github.com/redhat-helloworld-msa/frontend/commit/3cb22099f8fafd8007e0f07249e1f99cf054747d

**Pull request closed without merging**